sdk/deviceupdate/azure-iot-deviceupdate/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "java",
   "TagPrefix": "java/deviceupdate/azure-iot-deviceupdate",
-  "Tag": "java/deviceupdate/azure-iot-deviceupdate_6e7c02a488"
+  "Tag": "java/deviceupdate/azure-iot-deviceupdate_e6c0b7421b"
```

Comment:
Can we add the swagger used here https://github.com/chejiang-msft/azure-sdk-for-java/tree/user/chejiang/device-update-sync-stack/sdk/deviceupdate/azure-iot-deviceupdate/swagger#generate-autorest-code

Reply:
It would also be good add the autorest version used for the generation.
https://github.com/Azure/azure-sdk-for-java/blob/8c2577692d0b3900ddaee2dcf154ab6f8db30ca4/sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md?plain=1#L65

Reply:
How should I add the swagger and autorest version used? Like what key/value pair should I use? This is for the assets.json file not the yaml right?

Reply:
This is not regards to the assets.json file.

Since the swagger/readme file is not in the PR I commented here.
But it should be added in this section https://github.com/Azure/azure-sdk-for-java/pull/36905/files#r1343310754

Example: 
use: '@autorest/java@4.1.13'
input-file: swagger file

Reply:
I've added the config yaml file used